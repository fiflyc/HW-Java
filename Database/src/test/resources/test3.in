1 AAA 000
1 AAB 000
1 AAC 000
1 BBA 111
1 BBB 111
1 CCC 111
1 CCC 000
3 000
3 111
0