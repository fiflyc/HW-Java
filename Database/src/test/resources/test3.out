AAA AAB AAC CCC 
BBA BBB CCC 
